Makefile

```diff
@@ -36,7 +36,9 @@ deploy: manifests
 # Generate manifests e.g. CRD, RBAC etc.
 .PHONY: manifests
 manifests:
```

Comment:
adjust that for kubebuilder v2: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v2/Makefile

Reply:
I will stay with the kubebuilder v1 :(